### ISSUE TITLE
Report Major VA.gov Incidents Following VA OIT's Major Incident Management (MIM) Process 

### ISSUE THREAD

**Issue by raywangoctova** [department-of-veterans-affairs/va.gov-team#48270]

**Comment by raywangoctova** [comment]
Per discussion at the onsite workshop for 2023 platform objectives, Cory indicated that he would follow up on this project request for the next steps.

**Comment by little-oddball** [comment]
Per request:

There is some decent information around the request and with the general notion of Platform being compliant, that should add some clear / concrete boundaries to work within. As this ticket is currently written, there is a lot of unknown(s) from a Platform team perspective so a fair amount of discovery will be required to flush out the specifics, to get educated and get detailed on what items Platform needs and actions they should take to get there. This feels like a pretty good size effort and needs very clear MVPs identified and defined. I believe the AC is a bit high level and needs more added to it. For example, I would assume Platform documentation would need to be updated and communicated to Platform and VFS teams. I would also say conducting a tabletop exercise shouldn’t be an AC but making part of the process and possibly even scheduling the tabletop could (shouldn’t be holding the item success based on a future event that there is no control over). 

So to me, step #1 is to put a sprint of Discovery into play and work to iron out and harden the specifics, deliverables, etc. The output should be a clear path forward with very clear and specific AC with small bitesize and testable deliveries.

**Comment by jwoodman5** [comment]
I don't really understand this whole process, but do think I understand what the A/C should be. I feel like there would be fair bit of discovery that would happen along the way as the playbook is updates.

**Comment by AparnaNittalaUSDS** [comment]
This process is being formulated as part of the code Yellow, in collaboration with the ECC. Datadog monitor and ServiceNow integration is in place so that datadog alerts with Priority P1 (SEV-1 classification in SNOW) are available in ECC. Still there are few questions and processing pending discussion and process refinement on how many of these P1 alerts are to be handled throguh CPI/HPI MIM process of OIT.

**Comment by annekerr49** [comment]
Current status:
We have an initial POC to integrate Datadog monitors into SNOW for usage by ECC Event Management.
Next followup steps needed are:
Update alert message with required response action (contact Platform oncall? declare MIM?)
Configure alerting integration for all mission critical monitors (all P1/P2?)
Run tabletop exercise with ECO/EM/OTG to confirm successful incident response (make sure everyone has necessary access & understands procedures)

**Comment by AparnaNittalaUSDS** [comment]
@AparnaNittalaUSDS to check what has been done and what is remaining with the OIT integration.. outline items like documentation, tagging standards etc

**Comment by chrisj-usds** [comment]
Can anyone comment on where we are with this effort? 
@JeffKeeneVAGov @BillChapmanUSDS

**Comment by EWashb** [comment]
@va-albers @rjain21 this is a really old ticket that we could revamp in light of our current DataDog work, Platform support, and Watch Tower efforts

**Comment by rjain21** [comment]
Met ECC a while back and indicated that the first case we can implement is to check if va.gov is not showing up due to any reason (e.g. CSS hids it, offscreen etc). ECC mentioned Prasad would be building a Synthetic Monitor. I haven't heard back since then.

**Comment by chrisj-gov** [comment]
@rjain21 

> I haven't heard back since then.
Can you do a follow up and ask if this is still something they are planning to build?

**Comment by rjain21** [comment]
@chrisj-gov - I did talk to ECC (Jeffery). It seems they are backlogged. I will follow up again.

**Comment by rjain21** [comment]
@mchelen-gov Correct. Prasad has created a cloned version of the Synthetic (I added a few checks...mainly CSS specific). The plan is to monitor the Synthetic for a while (Not sure what the timing is. ECC (Jeffery) Proposed this path. In parallel Rajesh to work with ECC t start filling out SOPs

**Comment by EWashb** [comment]
I think we have some duplicative work going on that could likely be better represented if we create new tickets around the automated processes and also manual processes of running an incident. I know that Rajesh has a ton of info on the the ECC and synthetic pieces of this. I have been coordinating with our platform teams to get our internal processes together. I'll start a new ticket and then we can parse out the work.

There's likely also things to add from Security Team's Tabletop and Disaster Recovery Planning.

**Comment by EWashb** [comment]
Related to the process/playbook work ticketed here: https://github.com/department-of-veterans-affairs/digital-experience-products/issues/1491

I suggest we make sister tickets for @rjain21's efforts for "Expand and Automate Monitoring for DE Major Incident Detection" focused on:

- Expand synthetic monitoring to cover critical Veteran workflows across web and mobile.
- Apply monitor tagging standards consistently across DE.
- Ensure DataDog to SNOW pipeline creates structured events with key metadata.
- Define and test auto-escalation rules for high-severity outages.
- Build DE dashboards for monitoring state and detection metrics.
- Validate ECC integration for visibility and handoff.

Does this track with everyone's mental model? If so, I'll scope that out.
